### DISCUSSION TITLE
STM32H5 driver support

### DISCUSSION THREAD

**Discussion by egoltzman** [FreeRTOS/FreeRTOS-Plus-TCP#1132] [Q&A]
@bjsowa Can you share the test project for the Nucleo-H563?

**Comment by bjsowa** [comment]
I don't have such "test" project. I only tested it as part of a larger project which was initially written for Nucleo-F7, later ported for Nucleo-H563 and now runs on a custom board with STM32H563VIT6. The project is closed-source for now.

**Reply by egoltzman** [reply]
Thank you @bjsowa  for your answer.
I need to use it with 3.1.0 and I was told to use ipconfigIPv4_BACKWARD_COMPATIBLE, I hope this will work for me

**Reply by bjsowa** [reply]
AFAIK the main branch of FreeRTOS-Plus-TCP does not support STM32H5 for now. To use it with H5 series, you will need the STM32 network interface driver developed by @HTRamsey, available in #804 which is still a draft.
